crates/daemon/src/main.rs

```diff
@@ -37,6 +37,13 @@ mod rpc_session;
 mod sys_ctrl;
 mod tasks_fjall;
 
+#[cfg(not(target_env = "msvc"))]
+use tikv_jemallocator::Jemalloc;
```

Comment:
I'd prefer the allocator stuff in a separate commit, and we should run the performance numbers with and without it to compare.

There are many directions the allocator stuff could go in, and I'm not 100% on this specific one.